reference/glossary.md

```diff
@@ -0,0 +1,17 @@
+(reference-glossary)=
+# Glossary
+
+```{glossary}
+
```

Comment:
We should also have Azure here since our "Initialize cloud environment" how-to mentions Azure and LXD, but not AWS.